llvm/include/llvm/CAS/CASDB.h

```diff
@@ -172,17 +135,13 @@ class CASDB : public CASIDContext {
   virtual Expected<CASID> parseID(StringRef ID) = 0;
 
   /// FIXME: Remove these.
-  Expected<LeafNodeProxy> createBlob(StringRef Data);
-  Expected<TreeProxy> createTree(ArrayRef<NamedTreeEntry> Entries = None);
-  Expected<NodeProxy> createNode(ArrayRef<CASID> References, StringRef Data);
+  Expected<ObjectProxy> create(ArrayRef<ObjectRef> Refs, StringRef Data);
+  virtual Expected<ObjectHandle> store(ArrayRef<ObjectRef> Refs,
```

Comment:
I haven't given much thoughts about those names. Now it looks like `create` gives back `Proxy` and `store` gives back `Handle`. `store` is also almost never used in current code base.

I should remove the FIXME since the comment was about `createTree` and `createNode` and they are all gone.